src/lib/mongoBackend/TriggeredSubscription.cpp

```diff
@@ -117,7 +120,18 @@ std::string TriggeredSubscription::toString(const std::string& delimiter)
   std::stringstream ss;
 
   ss << throttling << delimiter << lastNotification << delimiter << formatToString(format) << delimiter << reference;  
-  ss << expression.q << delimiter << expression.georel << delimiter << expression.coords << delimiter << expression.geometry << delimiter;
+  ss << expression.georel << delimiter << expression.coords << delimiter << expression.geometry << delimiter;
 
   return ss.str();
 }
+
+
+
+/* ****************************************************************************
+*
+* TriggeredSubscription::stringFilterSet - 
+*/
+void TriggeredSubscription::stringFilterSet(StringFilter* stringFilterP)
+{
+  stringFilter = *stringFilterP;
```

Comment:
I understand that this method was the one thought for copying... but maybe I'm wrong.